src/coverlet.collector/coverlet.collector.csproj

```diff
@@ -15,6 +15,7 @@
     <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);PackBuildOutputs</TargetsForTfmSpecificContentInPackage>
     <!-- Open issue https://github.com/NuGet/Home/issues/8941 -->
     <NoWarn>$(NoWarn);NU5127</NoWarn>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Comment:
we don't have good explanation and we're a tool so for the moment I would remove the documentation generation and the warn fix CS1591